### ISSUE TITLE
telomere-nuclear envelope anchor activity

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2785]
+[Term]
+id: GO:0140473
+name: telomere-nuclear envelope anchor activity
+namespace: molecular_function
+def: "The binding activity of a molecule that brings together the telomeric region of a chromosome and the inner nuclear membrane by interacting with both the telomere and the nuclear membrane, in order to establish and maintain the telomeric location." []
+synonym: "nuclear envelope-telomere anchor activity" EXACT []
+synonym: "telomere-nuclear envelope anchoring activity" EXACT []
+is_a: GO:0043495 ! protein-membrane adaptor activity
+property_value: term_tracker_item #19229 xsd:anyURI
+created_by: pg
+creation_date: 2020-06-19T15:44:44Z
+

(some anchor at the telomere and some at the subtelomeric heterochromatin)

**Comment by ValWood** [comment]
Systematic ID | Gene name | Product description
-- | -- | --
SPBC19C7.10 | bqt4 | bouquet formation protein Bqt4
SPBC11B10.09 | cdc2 | cyclin-dependent protein kinase Cdk1/Cdc2
SPAC664.01c | swi6 | heterochromatin (HP1) family chromodomain protein Swi6
SPAC18G6.10 | lem2 | LEM domain nuclear inner membrane protein Heh1/Lem2
SPCC338.17c | rad21 | mitotic cohesin complex, non-SMC subunit Rad21 (kleisin)
SPAC16A10.07c | taz1 | shelterin complex subunit Taz1
SPBC1778.02 | rap1 | shelterin complex telomere binding subunit Rap1

**Comment by ValWood** [comment]
This term still hasn't appeared. Is GO updating OK @kimrutherford

**Comment by mah11** [comment]
No problem at our end; that term isn't in the latest downloadable GO files.

**Comment by kimrutherford** [comment]
There has been no snapshot in a few days.

You can check the date in this file: http://snapshot.geneontology.org/summary.txt

**Comment by kimrutherford** [comment]
You can also check here, which has copies of the OBO files used for the most recent Chado load:
https://curation.pombase.org/dumps/latest_build/ontologies/

The dates are also on the secret internal details page on the main site: https://www.pombase.org/internal-details

**Comment by ValWood** [comment]
OK reported, thanks

**Comment by kimrutherford** [comment]
There has been a GO snapshot (dated 2020-07-03), but there's no sign of "telomere-nuclear envelope anchor activity".

**Comment by mah11** [comment]
The ticket with the new GO term details is still open (https://github.com/geneontology/go-ontology/issues/19229). There's also a branch for that ticket that looks like it's still active/not merged into master, and there's one merged pull request for a different part of the ticket.

tl;dr: still waiting for GO; no evidence of anything wrong on our end

**Comment by kimrutherford** [comment]
That term is in Canto now.  The GO issue is still open though.

**Comment by ValWood** [comment]
that's ok, I can continue at pombase end in sept...

**Comment by ValWood** [comment]
see
 https://github.com/pombase/curation/issues/2658
